src/relax/op/tensor/binary.cc

```diff
@@ -78,6 +78,14 @@ StructInfo InferStructInfoBroadcastCMP(const Call& call, const BlockBuilder& ctx
          const TensorStructInfo& x2_sinfo) { return DataType::Bool(); });
 }
 
+InferLayoutOutput InferLayoutBinaryEwise(const Call& call,
+                                         const Map<String, Array<String>>& desired_layouts,
+                                         const VarLayoutMap& var_layout_map) {
+  ICHECK(NoDesiredLayout(call, desired_layouts));
+  Layout layout = GetLayout(var_layout_map, call->args[0]);
+  return InferLayoutOutput({layout, layout}, {layout}, Attrs(call->attrs));
```

Comment:
Don’t we take `args[1]` into account :eyes: ?

Reply:
If they have the same layout, then it makes no difference.

If they have different layout, then it doesn't matter if we transpose 0 or 1.